src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -743,6 +747,45 @@ def _get_workspace_resource_id(
         # this parameter does not need validation
         return workspace_resource_id
 
+    def get_ip_families(self) -> Union[List[str], None]:
+        """IPFamilies used for the cluster network.
+
+        :return: List[str] or None
+        """
+        return self._get_list_attr('ip_families')
+
+    def get_pod_cidrs(self) -> Union[List[str], None]:
+        """Obtain the CIDR ranges used for pod subnets.
+
+        :return: List[str] or None
+        """
+        return self._get_list_attr('pod_cidrs')
+
+    def get_service_cidrs(self) -> Union[List[str], None]:
+        """Obtain the CIDR ranges for the service subnet.
+
+        :return: List[str] or None
+        """
+        return self._get_list_attr('service_cidrs')
+
+    def _get_list_attr(self, param_key) -> Union[List[str], None]:
+        param = self.raw_param.get(param_key)
+
+        if param and isinstance(param, str):
+            return param.split(',')
+
+        return None
+
+    def get_load_balancer_managed_outbound_ipv6_count(self) -> Union[int, None]:
+        """Obtain the expected count of IPv6 managed outbound IPs.
+
+        :return: int or None
+        """
+        count_ipv6 = self.raw_param.get(
+            'load_balancer_managed_outbound_ipv6_count')
+
+        return count_ipv6 if count_ipv6 else None
```

Comment:
In this way, we would convert `0` to `None`. If this is expected, please add a unit test case to cover this.

Reply:
good catch. changed to return 0